olcut-core/src/main/java-templates/com/oracle/labs/mlrg/olcut/OLCUT.java

```diff
@@ -32,15 +32,39 @@
  * This class stores the current OLCUT version, along with other compile time information.
  */
 public final class OLCUT {
+    /**
+     * OLCUT version string.
+     */
     public static final String VERSION = "${project.version}";
 
+    /**
+     * OLCUT build timestamp.
+     */
     public static final String BUILD_TIMESTAMP = "${maven.build.timestamp}";
 
+    /**
+     * OLCUT major version number.
+     */
     public static final int MAJOR_VERSION;
+
+    /**
+     * OLCUT minor version number.
+     */
     public static final int MINOR_VERSION;
+
+    /**
+     * OLCUT point release number.
+     */
     public static final int POINT_VERSION;
+
+    /**
+     * OLCUT version tag.
+     */
     public static final String TAG_VERSION;
 
+    /**
+     * Is this OLCUT build a unreleased snapshot?
```

Comment:
If I were feeling pedantic, I might point out that this should be "an unreleased" rather than "a unreleased".  If I were feeling pedantic, of course.

Reply:
I'll fix it.